src/ui/units/wizard/constants/dialogColor.ts

```diff
@@ -25,4 +26,10 @@ export const DEFAULT_GRADIENT_STATE = {
     middleThreshold: undefined,
     rightThreshold: undefined,
     gradientPalette: DEFAULT_TWO_POINT_GRADIENT,
+    nullMode: undefined,
```

Comment:
why is it not equal to an DEFAULT_NULL_MODE here?

Reply:
nullMode should be undefined by default, because it is applicable not for all types of charts